app.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "sinatra/base"
+require "./conda_repo"
+require "builder"
+
+redis = Redis.new(url: ENV["REDIS_SERVER"], driver: :hiredis)
+CONDA_REPO = CondaRepo.new(redis)
+CONDA_REPO.update_packages
```

Comment:
This will be being removed in the next PR where I add a cron-like/[rssparsing script to run this every time a new rss feed entry has come up]